google-map-marker.html

```diff
@@ -366,7 +366,7 @@
         subtree: true
       });
 
-      var content = this.innerHTML.trim();
+      var content = this.children[0];
```

Comment:
@ebidel Sorry, I've been busy with other projects and haven't been able to get back to these until now.

@mlisook is correct that the div was added so that you could depend on this.children[0] containing all of the contents. I can look into whether there's a way to make it work with just a text node and fix the `if (content)` predicate.

@mlisook, Not sure I understand what your theory is on why the ink elements don't work in the InfoWindow. What's the fundamental problem you see with moving the DOM node?

Is there another possible way to preserve the events handlers attached to the contents of the marker besides this idea? Realistically, the google-maps-marker element is radically less useful without this ability.